Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -165,18 +169,19 @@ - (void)timeout:(NSTimer *)timer
 
 #pragma mark - Public API
 
-RCT_EXPORT_METHOD(startObserving:(RCTLocationOptions)options)
+RCT_EXPORT_METHOD(startObserving:(NSDictionary*)options)
```

Comment:
RCTLocationOptions is a C Struct, which to my knowledge cannot hold NSString values. I thought about assigning integer values to the permission types, or making RCTLocationOptions a class but this seemed like unnecessary complexity & change.